code/modules/species/species.dm

```diff
@@ -865,9 +865,9 @@ var/global/const/DEFAULT_SPECIES_HEALTH = 200
 		if(31 to 45)	. = 4
 		else			. = 8
 
+/**Runs extra processing after any organs member of this species goes through rejuvenation. */
```

Comment:
Why the block comment syntax for a single line comment?

Reply:
Cause this is the only comment format that properly shows proc description in the auto-complete.